Resources/ArmTemplates/parameters.json

```diff
@@ -0,0 +1,87 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "functionAppPrefix": {
+      "value": "__functionAppPrefix__"
+    },
+    "apimResourceGroup": {
+      "value": "__SharedResourceGroupName__"
+    },
+    "apimServiceName": {
+      "value": "__ApimServiceName__"
+    },
+    "apiName": {
+      "value": "__apiName__"
+    },
+    "apiProductDisplayName": {
+      "value": "__apiProductDisplayName__"
+    },
+    "apiSuffix": {
+      "value": "__apiSuffix__"
+    },
+    "appServicePlanName": {
+      "value": "__providerPortalAppServicePlan__"
+    },
+    "appServicePlanResourceGroup": {
+      "value": "__providerPortalResourceGroup__"
+    },
+    "storageConnectionString": {
+      "value": "__providerPortalStorageConnectionStr__"
+    },
+    "cosmosDBfqdn": {
```

Comment:
There's a mismatch between the parameter name and the variable used to tokenise the value.  Is that the correct variable?

Reply:
I've just double checked from the override parameters, and it's correct.